lib/RAST_SDK/RAST_SDKImpl.pm

```diff
@@ -1490,7 +1498,7 @@ sub annotate_genomes
 	#	4. Use perl grep to see if the ref is already in the list
 	#	5. Issue a warning when a duplicate is found so user knows what happened. 
 	#
-	if (ref $genomes eq 'ARRAY') {
+	if ( ref $genomes eq 'ARRAY' && @$genomes ) {
```

Comment:
May as well use `is_arrayref` for this too -- `if ( is_arrayref $genomes && @$genomes )`

Reply:
To avoid installing Ref::Util in Dockerfile, I rolled back to use the ref eq 'ARRAY' check instead of is_arrayref.

Reply:
See the change I made to Chris's fba_tools Dockerfile to use a `cpanfile` - it is a *much* better way to maintain perl modules than loading modules directly in the Dockerfile. It's analogous to the python `requirements.txt`.